src/App/DsiplayUsers/DisplayUsers.tsx

```diff
@@ -0,0 +1,106 @@
+import { Container, List, ListItem, makeStyles, Theme, Typography } from '@material-ui/core'
+import { useContext, useEffect, useState } from 'react'
+import { GlobalContext } from '../../GlobalContext'
+import { theme } from '../../theme'
+import { GetUsersEndpointResult } from '../../services/ApiService'
+
+function DisplayUsers() {
+	const { api, setTitle } = useContext(GlobalContext) // for page title and API access
+
+	const useStyles = makeStyles({
+		errorContainer: (theme: Theme) => ({
+			display: 'flex',
+			justifyContent: 'center',
+			marginTop: theme.spacing(3),
+		}),
+		resultContainer: (theme: Theme) => ({
+			display: 'flex',
+			flexDirection: 'row',
+			justifyContent: 'center',
+			marginTop: theme.spacing(3),
+		}),
+		resultRow: (theme: Theme) => ({
+			alignSelf: 'stretch',
+			border: '1px solid',
+			borderCollapse: 'collapse',
+			borderColor: theme.palette.primary.main,
+			display: 'flex',
+			justifyContent: 'space-between',
+			marginBottom: theme.spacing(3),
+			whiteSpace: 'nowrap'
+		}),
+	})
+	const classes = useStyles(theme)
+
+	const [users, setResult] = useState<GetUsersEndpointResult[]>([])
```

Comment:
By convention, these setter functions usually match the name of the variable they update

```suggestion
	const [users, setUsers] = useState<GetUsersEndpointResult[]>([])
```

Reply:
I should have seen that one